tests/modules/groups/test_groups.py

```diff
@@ -54,23 +54,25 @@ def test_get_group_positions_data(client):
 def test_get_position_holders(client):
     res = helpers.get_position_holders(5)
     assert len(res) == 2
-    assert res[0]["first_name"] == "Sean"
-    assert res[1]["first_name"] == "Robert"
+    res = [res[0]['first_name'], res[1]['first_name']]
+    assert 'Robert' in res and 'Sean' in res
```

Comment:
Could `assert set(row['first_name'] for row in res) == set(['Robert', 'Sean'])`. Ditto for others.

Reply:
Thanks, will use going forward